extend.h

```diff
@@ -1,3 +1,12 @@
+#ifndef EXTEND_H
+#define EXTEND_H
+
+#include <stddef.h>
+
+#include "mpool.h"
```

Comment:
Yes, `mpool.h` does not needed there. However `conev.h` is required for `struct poolhd` in function prototypes and `proxy.h` required for `struct sockaddr_ina` because they are defined in these headers. 
clangd gives warnings about this.

Reply:
ok, but conev.h is already in the proxy.h, but that’s nothing.